tests/generators_tests/test_procedure_generators.py

```diff
@@ -3,7 +3,7 @@
 
 
 def test_ProcedureGenerator():
-    value_set = [x["code"] for x in ProcedureCodeSimple().value_set]
+    value_set = [x.code for x in ProcedureCodeSimple().value_set]
```

Comment:
This looks great!  I noticed that our existing tests for the medication generators and condition generators don't check that the output is in the valueset in the same way that we do for procedure, could you modify the tests for the other generators to align with this?

Reply:
@adamkells Added check of outputs in valuesets. Look at the latest commit. Did I do it right or other changes needed.